packages/domain/source/server/todos/todo-list.js

```diff
@@ -1,8 +1,10 @@
 Space.eventSourcing.Aggregate.extend('Todos.TodoList', {
 
-  fields: {
-    name: String,
-    todos: [Todos.TodoItem]
+  fields() {
```

Comment:
Should revert this. see https://github.com/meteor-space/event-sourcing/pull/51#issuecomment-174206412